packages/pn-personafisica-webapp/.env.cypress

```diff
@@ -0,0 +1,16 @@
+REACT_APP_URL_FE_LOGIN='https://portale-login.dev.pn.pagopa.it/'
+REACT_APP_URL_API='https://webapi.dev.pn.pagopa.it/'
+REACT_APP_CHECKOUT_URL='https://uat.checkout.pagopa.it'
+REACT_APP_PAYMENT_DISCLAIMER_URL='https://www.pagopa.it'
+HOST='portale.dev.pn.pagopa.it'
+WEBAPP_URL='https://portale.dev.pn.pagopa.it:443'
+REACT_APP_MIXPANEL_TOKEN='DUMMY'
+REACT_APP_ONETRUST_DOMAIN_ID='fd5aef6f-a6d3-422b-87b7-aa5e2cb6510c'
+REACT_APP_PAGOPA_HELP_EMAIL='notifichedigitali@assistenza.pagopa.it'
+REACT_APP_VERSION=$npm_package_version
+REACT_APP_DISABLE_INACTIVITY_HANDLER=true
+REACT_APP_ONE_TRUST_PORTAL_CDN_PP='https://privacyportalde-cdn.onetrust.com/77f17844-04c3-4969-a11d-462ee77acbe1/privacy-notices/draft/365c84c5-9329-4ec5-89f5-e53572eda132.json'
```

Comment:
@maurizioflauti a lot of this env variables are going to change with this pr https://github.com/pagopa/pn-frontend/pull/584

Reply:
ok